File: SportsProject/src/main/java/ssv/com/entity/Schedule.java
package ssv.com.entity;

import java.sql.Date;
import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
public class Schedule {
	private int idSchedule;
	private Date timeEnd;
	private Date timeStart;
	private int idTeam1;
	private int idTeam2;
	private int scoreTeam1;
	private int scoreTeam2;
	private String address;
	private int idTour;
	private int idwinner;
	private String video;
	private int status;
	private String description;
	private String nameTour;
	private String title;

	private String image;

}

File: SportVueClient/src/models/schedule.js
export default class Schedule {
    constructor( timeStart,timeEnd,scoreTeam1,scoreTeam2, idTeam1,idTeam2,idwinner,video,description,address,status,idTour,idSchedule,fileVideo,fileImage,title) {
      this.timeStart = timeStart;
      this.scoreTeam1=scoreTeam1;
      this.scoreTeam2=scoreTeam2;
      this.timeEnd = timeEnd;
      this.idTeam1=idTeam1;
      this.idTeam2=idTeam2;
      this.idwinner=idwinner;
      this.video=video;
      this.description=description;
      this.address=address;
      this.status=status;
      this.idTour=idTour;
      this.title=title;
      this.idSchedule=idSchedule;
      this.fileVideo=fileVideo,
      this.fileImage=fileImage
    }
  }
  